src/rfquack.h

```diff
@@ -660,7 +660,7 @@ void rfquack_setup() {
 
   delay(100);
 
-  rfquack_radio_setup();
+  rfquack_radio_setup(rfqRadio);
```

Comment:
Ideally, this could be a RadioLib instance, unless the thin wrapper is absolutely necessary. Maintaining the wrapper is a lot of work, and adds complexity (and size) to the code base. Of course, this is not always possible, but I'd avoid having 1) one super class and 2) N sub-classes, one per module, because RadioLib does that already.